app/routers/web/views.py

```
@@ -93,6 +93,14 @@ async def edit_storage_location_form(request: Request, location_id: int):
     )
 
 
+@router.get("/files/{inventory_id}", response_class=HTMLResponse)
+async def get_file_details(request: Request, inventory_id: int):
```

Ruff (0.15.10)

[warning] Unused function argument: `inventory_id`

(ARG001)